TuringAdvisoryGroup/tryrolljs
# Add token select component

## What's done

- Add TokenSelect component
- Make CircleImg cross-platform
- Add onChange prop to Select

## How to test

1. Run storybooks for all the platforms.
2. Check that the rendered result matches the screenshots below.
3. Check that the tokens are selectable.

## Tasks


## Demo (screenshots, recordings, etc.)

<img width="564" alt="Screen Shot 2022-10-26 at 14 20 18" src="https://user-images.githubusercontent.com/17337276/198026518-fbf65a82-bf22-49e8-8200-4c29c9a51dfc.png">
<img width="677" alt="Screen Shot 2022-10-26 at 14 24 52" src="https://user-images.githubusercontent.com/17337276/198026525-572bd7f9-b4cf-4e89-9b94-28263309fcf4.png">
<img width="1073" alt="Screen Shot 2022-10-26 at 14 29 20" src="https://user-images.githubusercontent.com/17337276/198026529-5a1e127f-942c-4c2b-8574-cfec897cd79b.png">
